spec/simple_segment/configuration_spec.rb

```diff
@@ -22,6 +22,11 @@
       config = described_class.new(write_key: 'test')
       expect(config.on_error).to be_a(Proc)
     end
+
+    it 'has a default http_options' do
+      config = described_class.new(write_key: 'test')
+      expect(config.http_options).to eq({})
```

Comment:
@barodeur Could we move the default options to the Configuration class, this spec would look like:

```ruby
config = described_class.new(write_key: 'test')
expect(config.http_options).to eq({ http_options: { use_ssl: true } })
```

Sorry for nitpicking! :)

Reply:
no problem, I just moved the config to the configuration in 3ae2b4a